tests/product/test_status.py

```diff
@@ -48,11 +46,13 @@ def test_status_happy_path(self):
 
         # Test with worker not started
         self.run_prestoadmin('server start -H master')
-        status_output = self._server_status_with_retries()
+        # don't run with retries because it won't be able to query the
+        # coordinator because the coordinator is set to not be a worker
+        status_output = self.run_prestoadmin('server status')
         self.check_status(
             status_output,
             self.single_node_up_status(
-                ips, self.cluster.internal_master))
+                ips, self.cluster.internal_master, coordinator_down=True))
```

Comment:
sure. it seems like we always want to have this be true anyway, so I can clean it up and change that.